addons/mail/models/mail_thread.py

```diff
@@ -655,6 +653,99 @@ def _track_subtype(self, initial_values):
         self.ensure_one()
         return False
 
+    def _format_display_value(self, data, fieldinfo, author_id):
```

Comment:
I don't understand why it's more complex than the old _format_display_value implementation that delegates the formating to QWeb. Can't we just use the old implementation?